giftacart-backend/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::API
+  after_action :apply_content_range_header
+
+  protected
+  def apply_content_range_header
+    response.headers['Content-Range'] = 'orders 0-24/319'
```

Comment:
what's this for?

Reply:
React expects us to provide a content range in the response header so it knows how to paginate. This is a standard way of exposing that.